buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All

```diff
@@ -20,88 +20,258 @@
+*   VENDOR_CREDENTIALS_ID: String - the Jenkis credentials to connect to the vendor Git repository if VENDOR_REPO is a private repository
+*   LABEL: String - the node label(s) to run this job on; could be any node that has Git installed on it
+*   BUILD_NODE_LABELS: String - the labels of a node to compile and build the Eclipse OpenJ9 extensions for OpenJDK
+*   TEST_NODE_LABELS: String - the labels of a node to run tests on
+*
+*   Node labels could be a single label or node name or a boolean expression(e.g. hw.arch.x86 && sw.os.windows)
+*   Expected value for multiple platforms builds: platform.1=labels.1,platform.2=labels.2,...,etc
+*   e.g. aix_ppc-64_cmprssptrs=csp70027,linux_x86-64=(ci.project.openj9 && hw.arch.x86 && sw.os.ubuntu.14)
+*   Expected value for single platforms builds: label (no platform name required), e.g. csp70027
+*
+*   PERSONAL_BUILD: Choice: true, false - Indicates if is a personal build or not
+*/
 
+RELEASES = ['8', '9', '10']
 
-def OPENJDK_REPOS = ['8': 'https://github.com/ibmruntimes/openj9-openjdk-jdk8.git',
-                     '9': 'https://github.com/ibmruntimes/openj9-openjdk-jdk9.git',
-                     '10': 'https://github.com/ibmruntimes/openj9-openjdk-jdk10.git']
-def OPENJDK_BRANCHES = ['8': 'openj9',
-                        '9': 'openj9',
-                        '10': 'openj9']
+SPECS = ['aix_ppc-64_cmprssptrs'      : RELEASES,
+         'linux_390-64_cmprssptrs'    : RELEASES,
+         'linux_ppc-64_cmprssptrs_le' : RELEASES,
+         'linux_x86-64'               : RELEASES,
+         'linux_x86-64_cmprssptrs'    : RELEASES,
+         'win_x86'                    : ['8'],
+         'win_x86-64_cmprssptrs'      : RELEASES]
 
-def OPENJ9_REPO = 'https://github.com/eclipse/openj9.git'
-def OPENJ9_BRANCH = 'master'
+LABEL = params.LABEL
+if (!LABEL) {
+    LABEL = 'worker'
+}
 
-def OMR_REPO = 'https://github.com/eclipse/openj9-omr.git'
-def OMR_BRANCH = 'openj9'
+OPENJDK_REPO = [:]
+OPENJDK_BRANCH = [:]
+OPENJDK_SHA = [:]
 
-def builds = [:]
+builds = [:]
 
 timeout(time: 10, unit: 'HOURS') {
-    node('worker') {
+    timestamps {
+        node(LABEL) {
+            try {
+                checkout scm
+                def variableFile = load 'buildenv/jenkins/common/variables-functions'
+                def buildFile = load 'buildenv/jenkins/common/pipeline-functions'
 
-        checkout scm
-        buildFile = load 'buildenv/jenkins/common/pipeline-functions'
+                // parse variables file and initialize variables
+                variableFile.set_job_variables('wrapper')
+                def SHAS = buildFile.get_shas(OPENJDK_REPO, OPENJDK_BRANCH, OPENJ9_REPO, OPENJ9_BRANCH, OMR_REPO, OMR_BRANCH)
 
-        // fetch SHAs
-        OPENJ9_SHA = buildFile.get_sha(OPENJ9_REPO, OPENJ9_BRANCH)
-        OMR_SHA = buildFile.get_sha(OMR_REPO, OMR_BRANCH)
+                if (params.PLATFORMS) {
+                    // personal build, update build name
```

Comment:
Since we are using `PERSONAL_BUILD` param now this `if` should change

Reply:
fixed